contracts/reverseRegistrar/README.md

```diff
@@ -0,0 +1,9 @@
+# L2 Reverse Registrar
+
+## Summary
+
+The L2 Reverse registrar is a combination of a resolver and a reverse registrar that allows the name to be set for a particular reverse node.
+
+## Resolver Profiles
+
+Profiles are separate from resolvers as the authorised modifier needs the address, rather than the node
```

Comment:
Can you elaborate a bit more? I don't fully understand what it means. Also, isn't ETH and other addresses part of profile?